### DISCUSSION TITLE
How to I flash my GameShark Pro 3.3?

### DISCUSSION THREAD

**Discussion by jeremy64** [sanni/cartreader#414] [Need help]
I recently got a GameShark Pro v3.3 from eBay and I tried plugging it on my n64 the proper way and it didn't work. I cleaned it and tried to flash it but it stills doesn't work. Can anyone help me? ( It display the number 8 on the LCD if that helps )

**Comment by sanni** [comment]
Did the flashing succeed? If not what was the error?

**Reply by jeremy64** [reply]
I don't know if it got flashed but this it what it says after it finishes. ![20220719_022022](https://user-images.githubusercontent.com/13055380/179691488-bb2ac2ba-c4a9-4f7e-ae3c-64d00a2cc2dc.jpg)

This it was it says after the first pic. ![20220719_022039](https://user-images.githubusercontent.com/13055380/179691697-790e1e18-d7a8-4a8f-80f5-a079199ab101.jpg)

**Reply by jeremy64** [reply]
I'm using Super Mario 64 by the way.

**Reply by Nes-player4Life** [reply]
Is the rom you are flashing with a working good one? Or are you trying to restore the same broken firmware to itself?

**Comment by sanni** [comment]
Put the Gameshark into the Cart Reader without any game on top.
Flash it like you did before, that worked. But don't press a button when it tells you to press a button instead switch the Cart Reader off after "Verifying...ok".
Remove the Gameshark from the Cart Reader put Mario(flashing removes any previous boot code) on top, make sure it's perfectly centered as the Gameshark uses a wrong pitch cartridge slot and then plug the Gameshark/Mario tower into the N64 console.

**Reply by jeremy64** [reply]
Yep, that worked. I just had to turn it off just like you said, thank you!
